### DISCUSSION TITLE
Appeal to gateway.py

### DISCUSSION THREAD

**Discussion by GIFUS** [Rapptz/discord.py#7196] [Q&A]
How can I access the gateway.py, to the identify function from my bot project?
I'm trying to change the json payload $browser to Discord IOS to make the bot a mobile status
I tried changing the value directly in the gateway.py, everything works. But is it possible to do it more correctly? Because due to the fact that I change this in the library file, all my bots start to have the mobile status.
I will be grateful for good advice.

**Comment by lgaan** [comment]
Please note this is a confirmed discord bug and isn't supposed to be possible; doing so comes with risks, etc.

For that reason I don't believe anyone is going to help you with this.

**Reply by GIFUS** [reply]
I've seen it done on discord.js. My friend has been using this for a long time.
I want to do this for the sake of an experiment, no more.

**Reply by mesub7** [reply]
People have don't it but (as Logan said) because it's a discord bug, we cannot give support for it.

**Comment by Rapptz** [comment]
This isn't supported.
